### PR TITLE
Update kekda.json

### PR DESCRIPTION
**Pull request by yxshv**

### CODE REVIEW

File: domains/kekda.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "description": "A description",
-  "repo": "https://github.com/kekda-py/kekda-py.github.io",
+  "repo": "https://github.com/kekda-py/the-repo-is-private",
   "owner": {
-    "username": "kekda-py",
```
Comment (TheBotlyNoob) [inline domains/kekda.json:5]: Why did you change this? Your username is kekda-py

**Review by TheBotlyNoob — DISMISSED** [review]

File: domains/kekda.json (hunk 1 above)
Comment (kekda-py) [reply domains/kekda.json:5]: ohh wait...

Comment (kekda-py) [reply domains/kekda.json:5]: fixed

File: domains/kekda.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "description": "A description",
-  "repo": "https://github.com/kekda-py/kekda-py.github.io",
+  "repo": "https://github.com/kekda-py/the-repo-is-private",
```
Comment (MaskDuck) [inline domains/kekda.json:3]: just... remove... it?

**Review by MaskDuck — DISMISSED** [review]

File: domains/kekda.json (hunk 2 above)
Comment (kekda-py) [reply domains/kekda.json:3]: i can?
ok.. i am so dumb

**Review by mtgsquad — APPROVED** [review]
